content/blog/posts/o3de-ux-past-present-future.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Explore the Past, Present, & Future of O3DE's First-time User Experience"
+description: "Learn about how Amazon Web Services developed the first-time user experience for Open 3D Engine (O3DE) from Bhanuja Sanghavi, one of the UX designers contributing to O3DE."
+date: 2022-1-14
+slug: o3de-ftue-blog-post-1
+author: Bhanuja Sanghavi, Amazon Web Services
+blog_img: "/images/blog/announcement_thumbnail.jpg"
+full_img: "/images/blog/o3de-2111-release-blog-image.png"
+---
+
+Learn about how Amazon Web Services developed the first-time user experience for Open 3D Engine (O3DE) from Bhanuja Sanghavi, one of the UX designers contributing to O3DE.
+
+## Explore the Past, Present, & Future of O3DE's First-time User Experience
+
+​
+Hello, I'm Bhanuja! Since September 2020, I have had the opportunity to work as a User Experience (UX) Design Intern for Amazon on its game engine Lumberyard, the predecessor to Open 3D Engine (O3DE). Over the past year, I have worked closely with the UX, Engineering, and Product teams to research the engine's First-time User Experience (FTUE), as well as the subsequent designed solutions to improve the ease and efficiency with which users set up, learn, and build with O3DE.
+​
+From the instant game developers and content creators install the engine for the first time to the moment they begin to dive deep into O3DE's distinctive feature set, the FTUE is essential for shaping the user's path and, eventually, their success in building with O3DE. I'll explain what I mean.
```

Comment:
I believe it should be dive deep rather than deeply, because we're not talking about the degree of understanding but at the great depth of understanding.